### PR TITLE
Copy image to clipboard

### LINKED ISSUE #2582: Add option to copy image (not just link) *to* clipboard
For pasting page shots into github or irccloud (for example), it would be pretty nifty to have an option that just copied the image content directly to the clipboard.

### PR DESCRIPTION
**Pull request by chenba**
Fix #2582

### CODE REVIEW

File: locales/en-US/webextension.properties [hunk 1]
```diff
@@ -24,6 +24,9 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+copyScreenshot = Copy
```
Comment (flodolo) [inline locales/en-US/webextension.properties]: This string needs a localization comment, since it's unclear if it's a noun or a verb without context.

**Comment by johngruen** [comment]
@chenba gonna pull this down and check it out now

**Comment by johngruen** [comment]
@chenba 3 minor things:

1. For download only mode, keep Download as the default option
screen shot 2017-11-14 at 7 10 20 pm

2. For normal mode, flip save and copy
screen shot 2017-11-14 at 7 09 35 pm

3. The SVG fill color should match the other two...make `fill: #3e3d40`

**Comment by johngruen** [comment]
ALSO...left need a string change on the message

**Comment by chenba** [comment]
@johngruen Need to confirm: do you want to swap the download and copy buttons _only_ for download-only mode?  Or, we keep them consistent, always Cancel, Copy, Download [, Save]?

**Comment by chenba** [comment]
> ALSO...left need a string change on the message

@johngruen I don't understand this comment.  Left?

**Comment by chenba** [comment]
@johngruen Swapped the position of the copy and download buttons.  Take another look?

**Comment by ianb** [comment]
The code looks good, though for some reason the translations aren't working for me. I've even done a `make clean`. The translations are obviously there, but I get warnings like `Unknown localization message copyscreenshot`

**Comment by chenba** [comment]
@ianb Yeah, this is related to #3661.  If you load the addon through `about:debugging` then you'll definitely see the new l10n messages.

**Comment by ianb** [comment]
OK, we can merge (should also get squashed) once @johngruen gives his 👍

**Comment by chenba** [comment]
@flodolo Updated a couple of l10n messages in the latest commit (https://github.com/mozilla-services/screenshots/pull/3758/commits/8e33cb73f9cf8b6d9aa0a7ae09d85ba329479dec).  Could you take a quick look?

**Comment by eyaler** [comment]
is there a copy button also in the my shots library?
how about binding with ctrl-c?

**Comment by chenba** [comment]
@eyaler ctrl-c is in this patch.  There's no copy (to clipboard) button in My Shots; an issue's been filed for adding one to the page of an individual shot (#1646).  Could you file one for My Shots?

**Comment by eyaler** [comment]
here: https://github.com/mozilla-services/screenshots/issues/3792

**Pull request merged**